etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/model/odse/DebeziumMetadata.java

```diff
@@ -4,8 +4,12 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 import jakarta.persistence.Column;
 import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
 
 @Data
+@ToString
+@EqualsAndHashCode
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class DebeziumMetadata {
```

Comment:
Are these the only three metadata variable we require in this application?

Reply:
This is just a placeholder and we can add/delete/update as we proceed.